alicloud/table_alicloud_vpc_flow_log.go

```diff
@@ -0,0 +1,202 @@
+				Type:        proto.ColumnType_STRING,
+				Description: "The type of traffic that is captured.",
+			},
+
+			// Resource interface
+			{
+				Name:        "title",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.From(vpcFlowLogTitle),
+				Description: ColumnDescriptionTitle,
+			},
+			{
+				Name:        "akas",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     vpcFlowLogAkas,
+				Description: ColumnDescriptionAkas,
+				Transform:   transform.FromValue(),
+			},
+
+			// alicloud common columns
+			{
+				Name:        "region",
+				Description: ColumnDescriptionRegion,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("RegionId"),
+			},
+			{
+				Name:        "account_id",
+				Description: ColumnDescriptionAccount,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getCommonColumns,
+				Transform:   transform.FromField("AccountID"),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listVpcFlowLogs(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	region := plugin.GetMatrixItem(ctx)[matrixKeyRegion].(string)
+
+	// Create service connection
+	client, err := VpcService(ctx, d, region)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_flow_log.listVpcFlowLogs", "connection_error", err)
+		return nil, err
+	}
+	request := vpc.CreateDescribeFlowLogsRequest()
+	request.Scheme = "https"
+
+	response, err := client.DescribeFlowLogs(request)
+	if err != nil {
+		plugin.Logger(ctx).Error("alicloud_vpc_flow_log.listVpcFlowLogs", "query_error", err, "request", request)
+		return nil, err
+	}
+	for _, i := range response.FlowLogs.FlowLog {
+		plugin.Logger(ctx).Warn("alicloud_vpc.listVpc", "item", i)
+		d.StreamListItem(ctx, i)
+	}
+	return nil, nil
+}
+
+//// HYDRATE FUNCTIONS
+
+func getVpcFlowLog(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	region := plugin.GetMatrixItem(ctx)[matrixKeyRegion].(string)
+
+	// Create service connection
+	client, err := VpcService(ctx, d, region)
+	if err != nil {
+		plugin.Logger(ctx).Error("getVpcFlowLog", "connection_error", err)
+		return nil, err
+	}
+
+	var name string
+	if h.Item != nil {
+		log := h.Item.(vpc.FlowLog)
+		name = log.FlowLogName
+	} else {
+		name = d.KeyColumnQuals["name"].GetStringValue()
+	}
+
+	request := vpc.CreateDescribeFlowLogsRequest()
+	request.Scheme = "https"
+	request.FlowLogName = name
+	response, err := client.DescribeFlowLogs(request)
+	if err != nil {
+		plugin.Logger(ctx).Error("getVpcFlowLog", "query_error", err, "request", request)
+		return nil, err
+	}
+
+	if response.FlowLogs.FlowLog != nil && len(response.FlowLogs.FlowLog) > 0 {
+		return response.FlowLogs.FlowLog[0], nil
+	}
+
+	return nil, nil
+}
+
+//// TRANSFORM FUNCTIONS
+
+func vpcFlowLogAkas(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	i := h.Item.(vpc.FlowLog)
+	// Get project details
```

Comment:
Add new line before comment
```suggestion
	
	// Get project details
```